testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/QASettingsActivity.kt

```diff
@@ -308,6 +314,17 @@ class QASettingsActivity : BaseActivity() {
             settings.maxStorageSizeLimitInBytes = binding.edtMaxStorageLimit.text.toString()
         }
 
+        //Save color theme
+        var primaryColor = ""
+        var secondaryColor = ""
+        if (!binding.edtPrimaryColor.text.isNullOrEmpty()) {
+            primaryColor = binding.edtPrimaryColor.text.toString()
+        }
+        if (!binding.edtSecondaryColor.text.isNullOrEmpty()) {
```

Comment:
I think we need not add the null or empty check for secondaryColor, we can send the default #FFFFFF - white color, if this makes sense.

Reply:
i think we don't need to send a default color, just send a empty string if no value is set, sample app will handle that, It's just a check in the demo app if nothing is in settings just simply assign a empty string. thanks for you suggestion though.